docs/source/FAQ/team_management_migration.rst

```diff
@@ -0,0 +1,80 @@
+================================
+Teams User Management Migration
+================================
+
+Summary
+=======
+
+This is an FAQ about migrating your code to the :class:`~citrine.resources.team.Team` version of the Citrine Platform.
+
+What's new?
+====================
+
+Teams
+------
+
+The biggest fundamental change that affects Citrine Python when migrating to CP2 is the
+introduction of the Teams concept for access control. After this update, all projects and assets are owned by Teams that include one or more users on the platform. This concept was introduced to make it easier for Citrine Platform users to securely share assets across projects where it makes sense for their business.
+
+Projects
+---------
+Projects are now available as a Team asset. Once you have a Project in hand, methods related to asset sharing and user management will direct you to the corresponding Team methods, and other methods will continue to work as expected. 
```

Comment:
Drop "Once you have a Project in hand". You can just say "Project methods related to asset sharing..."